tools/ockam-hub-cli/src/auth/github.rs

```diff
@@ -0,0 +1,119 @@
+use colored::*;
+use std::collections::HashMap;
+use std::future::Future;
+use std::pin::Pin;
+use webbrowser;
+
+const GITHUB_CLIENT_ID: &'static str = "609323e5ba23958cb2f5";
```

Comment:
Can we externalize this as an environment variable?

Reply:
Or, if this value is a stable identifier that will absolutely never change, let's add documentation to say what the value is, how to obtain it. etc

Reply:
This value is a fixed value that represents the `ockam` app on github, it won't change.

Reply:
What do you think that is the best approch @jared-s ?

Reply:
Thanks for the clarification, It's okay for now.